### PR TITLE
`<Sidebar/>`- changing scrollbar colors

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Changing Sidebar scrollbar colors for both light and dark skins 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

File: src/Sidebar/test/Sidebar.visual.js [hunk 1]
```diff
@@ -106,3 +117,73 @@ storiesOf('Sidebar', module).add('Hidden', () => (
...
+            <Sidebar.Item itemKey={'mailboxes'}>
+              <SidebarSectionItem>Mailboxes</SidebarSectionItem>
+            </Sidebar.Item>
+
+            <SidebarDivider />
+
+            <Sidebar.Item itemKey={'hire-an-expert'}>
+              <SidebarSectionItem>Hire an Expert</SidebarSectionItem>
+            </Sidebar.Item>
+          </Sidebar>
```
Comment (nitayneeman) [inline src/Sidebar/test/Sidebar.visual.js]: If we're already testing the scrollbar, I would also add a simple persistent footer (not related to your changes but just to improve the domain this test covers)

File: src/Sidebar/test/Sidebar.visual.js [hunk 2]
```diff
@@ -106,3 +117,73 @@ storiesOf('Sidebar', module).add('Hidden', () => (
...
+
+            <Sidebar.Item itemKey={'mailboxes'}>
+              <SidebarSectionItem>Mailboxes</SidebarSectionItem>
+            </Sidebar.Item>
+
+            <SidebarDivider />
+
+            <Sidebar.Item itemKey={'hire-an-expert'}>
+              <SidebarSectionItem>Hire an Expert</SidebarSectionItem>
+            </Sidebar.Item>
```
Comment (nitayneeman) [inline src/Sidebar/test/Sidebar.visual.js]: Same feedback for the content - we can replace it with an element that higher than 500px

File: src/Sidebar/test/Sidebar.visual.js [hunk 3]
```diff
@@ -106,3 +117,73 @@ storiesOf('Sidebar', module).add('Hidden', () => (
...
+                  <Avatar size="size72" />
+                  <Box direction="vertical" align="center" marginTop={2}>
+                    <Text size="small" secondary weight="bold">
+                      Owner
+                    </Text>
+                    <Text size="tiny" secondary light weight="normal">
+                      owner@something.com
+                    </Text>
+                  </Box>
+                </Box>
```
Comment (nitayneeman) [inline src/Sidebar/test/Sidebar.visual.js]: I think we can simplify the header by replacing it with a shorter example (`div` with text for example).

The thing is that this test might fail from the wrong reason when SidebarHeader/TextButton/Box/Avatar/Text is broken - generally it's OK but that's not the purpose of **this** test so we would not get any benefit...

**Comment by TalZa1810** [comment]
Ayelet approved this. Waiting for tests to pass to merge.

**Pull request merged**